examples/plot_kernel_ridge_regression.py

```diff
@@ -54,7 +54,7 @@
 y = np.sin(X).ravel()
 
 # Add noise to targets
-y[::5] += 3 * (0.5 - rng.rand(X.shape[0]/5))
+y[::5] += 3 * (0.5 - rng.rand(int(X.shape[0]/5)))
```

Comment:
Unclear to me too. But for some reason, after I changed this, the `VisibleDeprecationWarning` was gone.